src/partials/editor.html

```diff
@@ -549,6 +549,48 @@ <h5>Popup labels</h5>
     </div>
 </div>
 
+<div class="gf-form-group">
+    <h5>Custom image overlay</h5>
+    <gf-form-switch class="gf-form" label="Enable" label-class="width-10" checked="ctrl.panel.enableOverlay"
+                    on-change="ctrl.refreshOverlay()">
+    </gf-form-switch>
+    <div class="gf-form" ng-show="ctrl.panel.enableOverlay == true">
+        <label class="gf-form-label width-10">Overlay URL</label>
+        <input type="url" class="input-small gf-form-input width-10" ng-model="ctrl.panel.overlayUrl"
+               ng-change="ctrl.refreshOverlay()"
+               ng-model-onblur/>
+    </div>
+    <div class="gf-form" ng-show="ctrl.panel.enableOverlay == true">
+        <label class="gf-form-label width-10">Overlay opacity</label>
+        <input type="number" step="0.05" class="input-small gf-form-input width-10" 
+                ng-model="ctrl.panel.overlayOpacity" placeholder="0.8"
+               ng-change="ctrl.refreshOverlay()"
+               ng-model-onblur/>
+    </div>
+    <div class="gf-form" ng-show="ctrl.panel.enableOverlay == true">
+        <label class="gf-form-label width-10">
+            Latitude range
+            <i class="grafana-tip fa fa-question-circle"
+               bs-tooltip="'Provide the min and max latitude values, separated by a comma'"></i>
```

Comment:
Are the additional single quotes needed here?

Reply:
As far as I can tell, yes, they are. It appears to interpret whatever is inside the attribute as JS (I can do `bs-tooltip="ctrl.panel.overlayLatRange"`, no single quotes, and it does display the value of the JS variable). The single quotes ensure that the text is treated as a string. Also, another help tooltip just below, from which I copied my code, does the same: see https://github.com/jreyesr/grafana-map-panel/blob/f1dd4d939a766116c13761409014ecc908c0d848/src/partials/editor.html#L600

Reply:
Ok, thanks! I didn't remember that detail.